src/extractor/scripting_environment_lua.cpp

```diff
@@ -31,13 +32,64 @@ template <> struct is_container<osmium::Node> : std::false_type
 template <> struct is_container<osmium::Way> : std::false_type
 {
 };
+template <> struct is_container<osmium::Relation> : std::false_type
+{
+};
 }
 
 namespace osrm
 {
 namespace extractor
 {
 
+class RelationsContainerWrap
```

Comment:
The wrapper is not needed and can be removed. Also `context.state.new_usertype<RelationsContainerWrap>( ...` can be removed. Relations can be forwarded to lua as

```
node_function(profile_table, node, result, sol::as_table(relations));
...
way_function(profile_table, way, result, sol::as_table(relations));sol::as_table(relations)
```